tests/test_cmdLineFwk.py

```diff
@@ -0,0 +1,216 @@
+# This file is part of ctrl_mpexec.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Simple unit test for cmdLineFwk module.
+"""
+
+import argparse
+import contextlib
+import pickle
+import os
+import tempfile
+import unittest
+
+from lsst.ctrl.mpexec.cmdLineFwk import CmdLineFwk
+from lsst.ctrl.mpexec.cmdLineParser import _PipelineAction
+import lsst.pex.config as pexConfig
+from lsst.pipe.base import (Pipeline, PipelineTask, PipelineTaskConfig,
+                            QuantumGraph, TaskFactory, InitInputDatasetField)
+import lsst.utils.tests
+
+
+@contextlib.contextmanager
+def makeTmpFile():
```

Comment:
Is there a reason you chose to make your own Context manager instead of using tempfile.TemporaryFile ?

Reply:
The reason is that I need the name of the file and not file object. It could potentially be done with `NamedTemporaryFile` but it still needs additional code to close the file, so I prefer old-fashioned  `mkstemp` instead in a separate context manager.